collector/collector.go

```diff
@@ -293,23 +295,24 @@ func (c collector) Collect(ch chan<- prometheus.Metric) {
 			result.Result.Status.GcodeMove.ExtrudeFactor)
 
 		// gcode position
-		ch <- prometheus.MustNewConstMetric(
-			prometheus.NewDesc("klipper_gcode_position_x", "Klipper gcode position X axis.", nil, nil),
-			prometheus.GaugeValue,
-			result.Result.Status.GcodeMove.GcodePosition[0])
-		ch <- prometheus.MustNewConstMetric(
-			prometheus.NewDesc("klipper_gcode_position_y", "Klipper gcode position Y axis.", nil, nil),
-			prometheus.GaugeValue,
-			result.Result.Status.GcodeMove.GcodePosition[1])
-		ch <- prometheus.MustNewConstMetric(
-			prometheus.NewDesc("klipper_gcode_position_z", "Klipper gcode position Z axis.", nil, nil),
-			prometheus.GaugeValue,
-			result.Result.Status.GcodeMove.GcodePosition[2])
-		ch <- prometheus.MustNewConstMetric(
-			prometheus.NewDesc("klipper_gcode_position_e", "Klipper gcode position for extruder.", nil, nil),
-			prometheus.GaugeValue,
-			result.Result.Status.GcodeMove.GcodePosition[3])
-
+		if len(result.Result.Status.GcodeMove.GcodePosition >= 4) {
```

Comment:
closing bracket in wrong location causing compile failure, should be `len(result.Result.Status.GcodeMove.GcodePosition) >= 4`